### ISSUE TITLE
FarmArk : Create Contact Section for Website

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/FarmArk#5]
Hey Hacktoberfest participants, good to see you.

This issue's goal is to create an external layout of contact page of the website.

Feel free to be creative! And please take care of the following criteria:

- In the page, add an embedded map with dummy location
- Include textual contact details along with contact form on to the right 

You can refer this link : http://cpconsol.com/contact-us.html

How to get it approved?

- Add a screenshot of  outlook on this thread and wait for one of the mentors to get back to you.
- Once it's approved, go ahead and make pr.

Please read [CONTRIBUTING.md](https://github.com/prathimacode-hub/FarmArk/blob/main/CONTRIBUTING.md) at least once before contributing code.

**Comment by aswinikalyan30** [comment]
Can I take this up as part of OpenCode?

**Comment by anj20** [comment]
hey! can u assign this to me??

**Comment by Isha307** [comment]
> hey! can u assign this to me??

Yes @anj20, I can assign it to you. But first, tell me your approach like what are you thinking and what tech stacks will you use.

**Comment by prathimacode-hub** [comment]
Explain your approach to get it assigned. @anj20

**Comment by anj20** [comment]
> > hey! can u assign this to me??
> 
> Yes @anj20, I can assign it to you. But first, tell me your approach like what are you thinking and what tech stacks will you use.

The contact section is divided into three section. First part, is the map part that provides the exact location of the office and the second part is the "Meet Us" section which includes a phone number and email for general inquiries, and a "Pitch Us" section which includes a template that helps farmers to communicate.

**Comment by prathimacode-hub** [comment]
Issue assigned to @anj20

**Comment by anj20** [comment]
![Screenshot (153)](https://user-images.githubusercontent.com/96625401/195979351-484b318c-f037-40d6-8fcb-3636ccf0ced1.png)

@Isha307 @prathimacode-hub  is this layout ok?

**Comment by prathimacode-hub** [comment]
This layout doesn't look professional. You can refer to some other websites for it. @anj20

**Comment by Isha307** [comment]
Any Update @anj20?

**Comment by prathimacode-hub** [comment]
Update please? @anj20

**Comment by prathimacode-hub** [comment]
Update please? @anj20

**Comment by codervivek5** [comment]
I want to work in this issue and I will do it within 1 day

**Comment by prathimacode-hub** [comment]
Ok, issue assigned to @codervivek5

**Comment by prathimacode-hub** [comment]
Assignment has been removed as there been no update from you yet. @anj20

**Comment by codervivek5** [comment]
@prathimacode-hub 
http://cpconsol.com/contact-us.html 

This link does not work.

**Comment by prathimacode-hub** [comment]
Ok, i think the link expired. That's ok. You can follow the generalized contact section which has contact details along with the map feature. @codervivek5
